### PR TITLE
Week 1

### PR DESCRIPTION
**Pull request by vm0123456789**

### CODE REVIEW

File: 01_OOP/01_object_oriented_piracy.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            crew (int): the count of crew on board
+        """
+        self.draft = draft
+        self.crew = crew
+
+    def is_worth_it(self) -> bool:
+        """is_worth_it calculates the weight of the ship without crew and
+        decides whether it is worthy to loot (if more than 20) or not.
+        """
+        return (self.draft - self.crew * 1.5) > 20
```
Comment (OrestSenkovskyi) [inline 01_OOP/01_object_oriented_piracy.py:23]: I like you one-line style, but please add some exception handling, for example draft will be a string

File: 01_OOP/03_color_ghost.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+class Ghost(object):
+
+    colors = ["white", "yellow", "purple", "red"]
+
+    def __init__(self) -> None:
+        """__init__ will set `color` attribute randomly choosing 
+        from the list of colors (class variable)
+        """
+        cls = self.__class__
+        self.color = random.choice(cls.colors)
```
Comment (OrestSenkovskyi) [inline 01_OOP/03_color_ghost.py]: Why do you need a class to get colors? You can use self.colors instead

File: 01_OOP/03_color_ghost.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# https://www.codewars.com/kata/53f1015fa9fe02cbda00111a/python
+
+import random
+
+
+class Ghost(object):
```
Comment (OrestSenkovskyi) [inline 01_OOP/03_color_ghost.py]: All new-style classes don't need to inherit from object

**Review by OrestSenkovskyi — COMMENTED** [review]
Please add examples of usage for every task

File: 01_OOP/01_object_oriented_piracy.py (hunk 1 above)
Comment (vm0123456789) [reply 01_OOP/01_object_oriented_piracy.py:23]: Added type check in the __init__

File: 01_OOP/03_color_ghost.py (hunk 2 above)
Comment (vm0123456789) [reply 01_OOP/03_color_ghost.py]: Replaced with instance variable

File: 01_OOP/03_color_ghost.py (hunk 3 above)
Comment (vm0123456789) [reply 01_OOP/03_color_ghost.py]: Yeah, I used codewars' template and it is some kind of legacy code

**Pull request merged**